src/tests/DevToys.Tests/Providers/TestData/Jwt/RS/PrivateKey.txt

```diff
@@ -0,0 +1,28 @@
+﻿-----BEGIN PRIVATE KEY-----
```

Comment:
I assume none of these keys are sensitive information to anyone, right?

Reply:
No, I generated them for testing purpose.